packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -525,8 +525,6 @@ class _ResidentWebRunner extends ResidentWebRunner {
       }
     } on Exception catch (err) {
       return OperationResult(1, err.toString(), fatal: true);
-    } on WipError catch (err) {
```

Comment:
👍 I'd seen this when trying local upgrades of `package: webkit_inspection_protocol`.

Reply:
That's because WipError implements exception now. Analyzer rules:)